mongo/integration/client_side_encryption_prose_test.go

```diff
@@ -2220,8 +2232,14 @@ func TestClientSideEncryptionProse(t *testing.T) {
 		}
 	})
 
-	rangeRunOpts := mtest.NewOptions().MinServerVersion("6.2").Topologies(mtest.ReplicaSet, mtest.Sharded, mtest.LoadBalanced, mtest.ShardedReplicaSet)
+	rangeRunOpts := mtest.NewOptions().MinServerVersion("7.0").Topologies(mtest.ReplicaSet, mtest.Sharded, mtest.LoadBalanced, mtest.ShardedReplicaSet)
```

Comment:
Optional: `rangeRunOpts` is a duplicate of `autoKeyRunOpts`. Is there a way to refactor this opts?